trade/trader_binance.js

```diff
@@ -151,37 +155,31 @@ module.exports = (apiKey, secretKey, pwd, log, publicOnly = false, loadMarket =
      * @param {Boolean} nonzero Return only non-zero balances. By default, Binance return zero assets as well.
      * @returns {Promise<unknown>}
      */
-    getBalances(nonzero = true) {
+    async getBalances(nonzero = true) {
       const paramString = `nonzero: ${nonzero}`;
 
-      return new Promise((resolve) => {
-        binanceApiClient.getBalances().then((data) => {
-          try {
-            let result = [];
-
-            data.balances.forEach((crypto) => {
-              result.push({
-                code: crypto.asset,
-                free: +crypto.free,
-                freezed: +crypto.locked,
-                total: +crypto.free + +crypto.locked,
-              });
-            });
-
-            if (nonzero) {
-              result = result.filter((crypto) => crypto.free || crypto.freezed);
-            }
+      try {
```

Comment:
For `async` implementation, don't use nested `try-catch`.
See `getOrderDetails()` as a sample.